app/uk/gov/hmrc/cardpaymentfrontend/views/ForceDeleteAnswersPage.scala.html

```diff
@@ -18,33 +18,39 @@
 @import uk.gov.hmrc.cardpaymentfrontend.views.html.Layout
 @import uk.gov.hmrc.govukfrontend.views.Aliases.{Button, Text}
 @import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
-@import uk.gov.hmrc.cardpaymentfrontend.actions.JourneyRequest
 
 @this(
         layout: Layout,
         govukButton: GovukButton
 )
 
 @(
-        statusMessageKey: String,
+        loggedIn: Boolean,
         redirectLink: Option[Url]
-)(implicit journeyRequest: JourneyRequest[_], messages: Messages)
+)(implicit request: Request[_], messages: Messages)
 
-@heading = @{messages(s"force-delete-answers.heading.${statusMessageKey}")}
+@statusMessageHeading = @{
```

Comment:
refactored to avoid using string interpolation for message keys, makes finding usage way easier.